stratify/_conservative.pyx

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+cimport numpy as np
+
+
+cdef calculate_weights(np.ndarray[np.float64_t, ndim=2] src_point,
+                       np.ndarray[np.float64_t, ndim=2] tgt_point):
+    # src_min    src_max
+    #    |----------|                 : Source
+    #       tgt_min     tgt_max
+    #          |------------|         : Target
+    #    |----------|                 : Delta (src_max - src_min)
+    #          |----|                 : Overlap (between src & tgt)
+    #    weight = overlap / delta
+    cdef Py_ssize_t src_ind, tgt_ind
+    cdef np.float64_t delta, weight
+    cdef np.ndarray[np.float64_t, ndim = 2] weights
+    cdef np.ndarray[np.float64_t, ndim = 1] src_cell, tgt_cell
+
```

Comment:
It could be cython convention that I'm not familiar with, but I'd expect these to be "ndim=2" without spaces?  Similarly for lines 35 and 36.